client/scss/components/_listing.scss

```diff
@@ -112,6 +112,11 @@ ul.listing {
   }
 
   &.chooser {
+    tbody .parent-page {
+      font-size: 1.15em;
+      background-color: #e6e6e6;
```

Comment:
also, this may need something like:

```scss
tbody .parent-page .title {
    a { display: inline-block; }
    .status-tag--label { border: 1px solid; }
}
```

to retain previous look

<details>
<summary>screenshots</summary>

Before this PR | This PR
---------------|----------------
![chooser-parent-page-title-row-before](https://user-images.githubusercontent.com/31622/169580428-7ab50dc5-599a-43a3-9d86-98c1ad75042e.png) | ![Screenshot 2022-05-20 at 18 20 50](https://user-images.githubusercontent.com/31622/169580451-cc0cf17a-8d5a-474b-afd5-3da42d347256.png)

</details>

Reply:
Added in [8e51fcf](https://github.com/wagtail/wagtail/pull/8485/commits/8e51fcf6b35dd77c2b9ef2419b5b609956b75cf0). To be honest I'm not sure how much of the existing styling of this row was really intentional - things like the font size seem to be remnants of the styling for the main listing page, and these status tags do show on a separate line without a border for the other rows in the table - but yep, since this is just meant to be a refactoring PR it makes sense to preserve them as closely as possible.